Makefile

```diff
@@ -70,3 +70,7 @@ build-site: build-playground ## Builds the whole site (including playground)
 .PHONY: build-blog
 build-blog: ## Builds the blog
 	cd blog && yarn && yarn build
+
+.PHONY: dev
+dev: ## Start docs dev server
```

Comment:
We have `docs:dev` command in `package.json`. Is it necessary to have it defined here too?

Reply:
I like consolidating common commands in a single file. But no, it's not necessary per se. Would you prefer that I remove it?

Reply:
I moved all npm scripts to Makefile in https://github.com/melange-re/melange-re.github.io/pull/189/commits/125f3efbae1646edce49b96bdeb79d383b7dc21f